src/Language/PureScript/Docs/RenderedCode/RenderKind.hs

```diff
@@ -1,4 +1,5 @@
 {-# LANGUAGE ScopedTypeVariables #-}
+-- TODO: Remove
```

Comment:
Were you intending to address this in this PR?

Reply:
Ah yeah. Kinds need to be removed, but I had issues with docs because I wasn't sure about compatibility maybe? I don't remember. I'll try removing it again and see where I got stuck.